cms/envs/common.py

```diff
@@ -898,6 +898,9 @@
     # programs support
     'openedx.core.djangoapps.programs',
 
+    # Catalog integration
```

Comment:
Do we need to have this integration on CMS? If we don't, I think we should not add the app to CMS.